pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -540,14 +515,17 @@ abstract contract LinearPool is ILinearPool, IGeneralPool, IRateProvider, BasePo
         }
     }
 
-    function _scalingFactors() internal view virtual override returns (uint256[] memory) {
+    /**
+     * @notice Return the scaling factors for all tokens, including the BPT.
+     */
+    function getScalingFactors() public view virtual override returns (uint256[] memory) {
```

Comment:
We can leave this as-is for now, but moving forward the scaling factors should not include BPT. We can create an issue for this and address it after merging here, to make it so this PR simply moves from one base pool to another.

Reply:
Created https://github.com/balancer-labs/balancer-v2-monorepo/issues/1960